gcp/pd-backed-nfs-server/main.tf

```diff
@@ -0,0 +1,46 @@
+provider "kubectl" {
```

Comment:
this repo is not the template repo - the user should use this module as is in his code. so we don't want to add a provider here. you can add the example to template repo with the provider

Reply:
But if you added it only for the dev step - אל תתייחס

Reply:
moved the providers to the readme